Cargo.toml

```diff
@@ -12,16 +12,16 @@ authors = ["Sean McArthur <sean.monstar@gmail.com>",
 keywords = ["http", "hyper", "hyperium"]
 
 [dependencies]
-cookie = "*"
 httparse = "*"
 log = ">= 0.2.0"
-mime = "*"
-openssl = "*"
-rustc-serialize = "*"
 time = "*"
-unicase = "*"
 url = "*"
 
+[dependencies.hypernet]
+path = "hypernet"
```

Comment:
I'm fairly confident this will prevent uploading to crates.io. We will have to upload and depend on hypernet and hyperprotocol as normal dependencies.